cumulusci/robotframework/Salesforce.py

```diff
@@ -757,6 +792,15 @@ def wait_until_salesforce_is_ready(self, locator=None, timeout=None, interval=5)
             self.builtin.log("waiting for a refresh...", "DEBUG")
             self.selenium.capture_page_screenshot()
             time.sleep(interval)
+            location = self.selenium.get_location()
+            if "//test.salesforce.com" in location:
+                # did we land on https://test.salesforce.com? If so, maybe
```

Comment:
Note that it would probably be login.salesforce.com if running tests against an org that's not on a sandbox instance.

Reply:
Thanks for the heads up.  This was initially a fix for the fact my machine was _always_ hitting the redirect Monday afternoon. I'll add a check for either 'login.salesforce.com' or 'test.salesforce.com'